chglog.go

```diff
@@ -324,18 +326,6 @@ func (gen *Generator) render(w io.Writer, unreleased *Unreleased, versions []*Ve
 		"datetime": func(layout string, input time.Time) string {
```

Comment:
We can remove all the funxs here but that would break backwards compatibility. Let me know.

Reply:
Right on. I'd like to do some testing to make sure.